public/restaurant.html

```diff
@@ -1,71 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Find Food!</title>
-
-    <!--GOOGLE FONT IMPORTS-->
-    <link
-      href="https://fonts.googleapis.com/css2?family=Concert+One&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300&display=swap"
-      rel="stylesheet"
-    />
-
-    <!-- LEAFLET CSS FILE-->
-    <link
-      rel="stylesheet"
-      href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
-      integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
-      crossorigin=""
-    />
-
-    <!-- LEAFLET JS FILE-->
-    <script
-      src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
-      integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
-      crossorigin=""
-    ></script>
-
-    <!--MY CSS STLYE SHEET-->
-    <link href="restaurant.css" rel="stylesheet" />
-  </head>
-  <body>
-    <!--MAIN HEADER-->
-    <div id="header">
-      <a id="yelpington" href="./index.html">
-        <h1>YELPINGTON</h1>
-        <h6>Find food in your area...</h6>
-      </a>
-    </div>
-    <!--DIV FOR MY RESTAURANT FLEXBOX!-->
-    <div id="mapflex">
-      <!--WHERE RESTAURANT DATA IS DISPLAYED!-->
-
-      <div id="restinfo">
-        <h2 id="name"></h2>
-        <br />
-        <p id="address"></p>
-        <br />
-        <p id="phone-num"></p>
-        <br />
-        <p id="hours"></p>
-        <br />
-        <p id="notes"><b>User Comments:</b> </b></p>
-        <br />
-      </div>
```

Comment:
if you were to set your CSS of restinfo to a `flex` and the `flex-direction` to be a `column` with `justify-content: space evently` you'll get the same effect without using `<br />` tags